nmostesting/suites/IS1101Test.py

```diff
@@ -1715,3 +1716,419 @@ def test_04_02_02(self, test):
             if len(self.caps["constraint_sets"]) == 0:
                 return test.WARNING("The receiver does not support BCP-004-01.")
         return test.PASS()
+
+    def test_04_03(self, test):
+        """
+        Verify receivers supporting outputs
+        """
+        if len(self.receivers) == 0:
+            return test.UNCLEAR("No IS-11 receivers")
+        for receiver_id in self.receivers:
+            _, response = TestHelper.do_request('GET', self.compat_url + "receivers/")
+            if response.status_code != 200:
+                return test.FAIL("The receiver's streamcompatibility request has failed: {}"
+                                 .format(response.json()))
+            self.receivers_outputs = response.json()
+
+            """
+            The test requires streaming from a Sender in order to
+            verify the state of the Receiver and the associated outputs.
+            """
+            """
+            if IS11_RefecenceSender is not configured fail 4.3 with log indicating
+            that IS11_RefecenceSender is not configured
+            """
+            if len(CONFIG.IS11_REFERENCE_SENDER) == 0:
+                return test.FAIL("IS11_RefecenceSender is not configured")
```

Comment:
Should use `test.DISABLED()` I think?

https://github.com/AMWA-TV/nmos-testing/blob/b818d3ab414c7df0c37540e223af9a99f856ffd0/docs/5.1.%20Extension%20-%20Adding%20New%20Tests.md?plain=1#L36-L38